lib/components/Dialog/DialogListItem.stories.tsx

```diff
@@ -170,7 +170,7 @@ export const Statuses = (args: DialogListItemProps) => {
   );
 };
 
-export const Loading = (args: DialogListItemProps) => {
+const Loading = (args: DialogListItemProps) => {
```

Comment:
Do we need to hide the story in order to skip the test when `skip-test` is used?